Makefile

```diff
@@ -6,6 +6,8 @@ PG_CONFIG   ?= pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 
+DATABASE_URL:=postgres:postgres@localhost:5432/postgres 
```

Comment:
```suggestion
```
We want to allow `make test` to use any database url, not a fixed database

Reply:
There's a default in the rust code. We may write a default here but I think we should allow it to be overriden

Reply:
This line is why extension upgrade tests fail, they are relying on a pgrx managed postgres and thus use a different `DATABASE_URL`

Reply:
I updated the Makefile. Take a look, we should be able to set a default AND allow it to be overridden.

Reply:
Ah I see, looks like we're setting it to [DATABASE_URL=postgres://$USER:postgres@localhost:28816/pgmq](https://github.com/tembo-io/pgmq/blob/c01dd3450d0a51ee5dd45b840db361ddcbc3b7b9/.github/workflows/extension_upgrade.yml#L92). This should work again.

Reply:
I see now. Can't go back in time and change the Makefile, so I'll just remove this part completely.